packages/pangea-sdk/tests/integration/asyncio/test_vault.py

```diff
@@ -626,3 +626,33 @@ async def test_folders(self):
         # Delete parent folder
         delete_resp = await self.vault.delete(id=create_parent_resp.result.id)
         self.assertEqual(delete_resp.result.id, create_parent_resp.result.id)
+
+    async def test_encrypt_structured(self):
+        key = await self.vault.symmetric_generate(
+            algorithm=SymmetricAlgorithm.AES256_CFB, purpose=KeyPurpose.ENCRYPTION, name=get_name()
+        )
+        self.assertIsNotNone(key.result)
+
+        data: dict[str, str | list[bool | str]] = {"field1": [1, 2, "true", "false"], "field2": "data2"}
+
+        encrypted = await self.vault.encrypt_structured(id=key.result.id, structured_data=data, filter="$.field1[2:4]")
+        self.assertIsNotNone(encrypted.result)
+
+        encrypted_data = encrypted.result.structured_data
+        self.assertIn("field1", encrypted_data)
+        self.assertEqual(len(data["field1"]), len(encrypted_data["field1"]))
+        self.assertEqual(data["field1"][0], encrypted_data["field1"][0])
+        self.assertEqual(data["field1"][1], encrypted_data["field1"][1])
+        self.assertNotEqual(data["field1"][2], encrypted_data["field1"][2])
+        self.assertNotEqual(data["field1"][3], encrypted_data["field1"][3])
+
+        self.assertIsIn("field2", encrypted_data)
```

Comment:
Shouldn't it be `assertIn()`?

Reply:
Oh, the asyncio tests are not running.
```
Installing the current project: pangea-sdk (3.5.0)
$ poetry run python -m unittest tests.integration.asyncio
----------------------------------------------------------------------
Ran 0 tests in 0.000s
OK
Cleaning up project directory and file based variables 00:00
Job succeeded
```

Reply:
Added an `__init__.py` for those tests but have to leave them disabled because they depend on a Python v3.8 API, and our CI is still on Python v3.7. Can revisit that with/after #178.